source/routing/get_stop_times.cpp

```diff
@@ -141,15 +141,19 @@ get_stop_times(const std::vector<routing::JppIdx>& journey_pattern_points,
 std::vector<std::pair<uint32_t, const type::StopTime*>>
 get_all_stop_times(const routing::JourneyPattern& jp,
                    const routing::JourneyPatternPoint& jpp,
-                   const std::string calendar_id,
+                   const std::string& calendar_id,
                    const type::VehicleProperties& vehicle_properties) {
 
     std::set<const type::MetaVehicleJourney*> meta_vjs;
     auto insert_meta_vj = [&](const nt::VehicleJourney& vj ) {
         assert(vj.meta_vj);
         meta_vjs.insert(vj.meta_vj);
     };
-    for (const auto* vj: jp.discrete_vjs) { insert_meta_vj(*vj); }
+    for (const auto* vj: jp.discrete_vjs) {
+        if(vj->isBaseScheduled()) {
```

Comment:
In a calendar mode, we don't want other freshness than base_shedule. We don't want real_time for a "days of scholar week".